<launch>

  <node name="image_view_income" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/>
    <!-- <remap from="image" to="/camera/rgb/image_rect_color"/>-->
    <param name="autosize" value="false" />
  </node>

  <node name="image_view_face" pkg="image_view" type="image_view" respawn="false" output="screen">
     <remap from="image" to="/face_detection/face_image"/> 
    <!-- <remap from="image" to="/face_detection/all_faces_image"/>-->
    <param name="autosize" value="false" />
  </node>
  
<rosparam file="/home/jsaraydaryan/ros_robotcupathome_ws/src/people_management/ros_face_recognition/config/common_face.yaml" />
  <node pkg="people_face_identification" type="PeopleFaceIdentification_simple.py" name="PeopleFaceIdentificationSimple"  output="screen" >
    
  </node>

</launch>